pkg/blobstore/readfallback/read_fallback_blob_access.go~

```diff
@@ -77,13 +81,21 @@ func (eh *readFallbackErrorHandler) OnError(observedErr error) (buffer.Buffer, e
 		return nil, util.StatusWrap(observedErr, "Secondary")
 	}
 	if eh.secondary == nil {
-		// Both backends returned NOT_FOUND. Don't bother
-		// prepending the name of the backend.
+		// No secondary exists, so just return the error
```

Comment:
Please end sentences source code comments and documentation with a full stop.

Reply:
.